### ISSUE TITLE
Any way to use Option[Ping]?

### ISSUE THREAD

**Issue by thunderkid** [mkotsur/aws-lambda-scala#15]
I'm trying to write a handler that can take an option, ie receive either a json string representing `Ping`, or a null/blank (which I'm actually going to use in a scheduler to fire every minute just to keep the lambda warm.) I have:

`class PingPongHandler extends Lambda[Option[Ping], Pong] { ...`

When I invoke with `{ "inputMsg": "whatever" }`, it's happy, and the handler receives a `Some`. However, I can't figure out what to invoke it with to generate a `None`. I've tried passing `null` which is how circe encodes a `None`, but that just gives circe decoding failure. So does `"null"`, `""`, `{}` etc. Is there some reason that options woudn't work with this library? Thanks!

**Comment by mkotsur** [comment]
Hm... I think it should work fine. In fact, I've just added a test (482a992), which checks that.

**Comment by thunderkid** [comment]
Ok, so I've implemented a test case similar to yours and it passes the tests fine. But these scalatests are working by running `PingPongHandler.handle` in the local development JVM. The problem I was having is after deployment to aws. I was trying to test it with 
`serverless invoke -f pingPongOptionFunc -d "null"`

(from powershell serverless cli) and that gave the error:

```
{
    "errorMessage": "Attempt to decode value on failed cursor: DownField(inputMsg)",
    "errorType": "io.circe.DecodingFailure$$anon$2",
    "stackTrace": []
}

  Error --------------------------------------------------

  Invoked function failed

     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Forums:        forum.serverless.com
     Chat:          gitter.im/serverless/serverless

  Your Environment Information -----------------------------
     OS:                     win32
     Node Version:           6.11.2
     Serverless Version:     1.26.1
`
```
I also tried putting the string `null` in a file and using `--path` instead but that gave the same result. Same with sticking in extra quotes. However, if I invoke it on a schedule from `serverless.yaml` using this:
```
functions:
  pingPongOptionFunc:
    handler: handler.PingPongOptionHandler::handle
    events:    
      - schedule:
          rate: rate(1 minute)
          input: "null"
          enabled: true
```
then I can check the logs and it appears to be behaving correctly. (And this is my actual use case.)
So perhaps this behavior is due to the way the serverless cli handles the string `"null".`, in which case it's probably outside the responsibility of the `aws-lambda-scala` library.

**Comment by thunderkid** [comment]
Yes, you're right, it seems to work perfectly from aws cli. It was just the serverless "feature" turning `null -> {}` that was throwing me off. Thanks.
